src/web/mjs/connectors/templates/WordPressMadara.mjs

```diff
@@ -88,23 +88,37 @@ export default class WordPressMadara extends Connector {
         // TODO: setting this parameter seems to be problematic for various website (e.g. ChibiManga, AniMangaES server will crash)
         uri.searchParams.set('style', 'list');
         let request = new Request(uri, this.requestOptions);
-        let data = await this.fetchDOM(request, this.queryPages);
+        let data;
+        if (this.queryPages instanceof Array) {
```

Comment:
CSS queries have support for or operator (`,`), so there is no need to iterate through multiple queries (this implementation would also break the page order in case the queries are alternating)